lib/vagrant-openshift/action/run_sti_tests.rb

```diff
@@ -34,7 +34,7 @@ def call(env)
           cmds = ['OUTPUT_COVERAGE=/tmp/sti/artifacts/coverage hack/test-go.sh']
 
           if @options[:all]
-            cmds << 'STI_TIMEOUT="--timeout 240s" hack/test-integration.sh'
```

Comment:
STI_TIMEOUT was renamed in source-to-image some time ago and this was missed, so it was a no-op.  S2I_TIMEOUT upstream is set by default to a sufficiently similar value (and things have been working fine anyway) that I figured removal was the best option.